src/client/java/teammates/client/scripts/DataMigrationForGoogleIdToGmail.java

```diff
@@ -0,0 +1,35 @@
+package teammates.client.scripts;
+
+import com.googlecode.objectify.cmd.Query;
+
+import teammates.storage.entity.Account;
+
+/**
+ * Script to set Google ID as its corresponding Gmail if it is not already.
+ */
+public class DataMigrationForGoogleIdToGmail extends GoogleIdMigrationBaseScript {
+
+    public static void main(String[] args) {
+        new DataMigrationForGoogleIdToGmail().doOperationRemotely();
+    }
+
+    @Override
+    protected boolean isPreview() {
+        return true;
+    }
+
+    @Override
+    protected Query<Account> getFilterQuery() {
+        return ofy().load().type(Account.class);
+    }
+
+    @Override
+    protected boolean isMigrationOfGoogleIdNeeded(Account account) {
+        return !account.getGoogleId().toLowerCase().endsWith("@gmail.com");
```

Comment:
This is not the right condition. A Google ID can end with some other domain. It's only when there is no domain that you need to append `@gmail.com` to it.

Reply:
Hm, may I know why is it that a Google ID can end with other domains? Is this a legacy thing?

Reply:
That's not legacy; that's how Google OAuth2 works.

Reply:
Ah I see, I didn't know that one can create a Google account using a non-Gmail email address.